thirdparty/build-thirdparty.sh

```diff
@@ -472,6 +472,7 @@ build_brpc() {
     LDFLAGS="-L${TP_LIB_DIR} -static-libstdc++ -static-libgcc" \
     ${CMAKE_CMD} -G "${GENERATOR}" -v -DBUILD_SHARED_LIBS=0 -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR \
     -DBRPC_WITH_GLOG=ON -DWITH_GLOG=ON -DCMAKE_INCLUDE_PATH="$TP_INSTALL_DIR/include" \
+    -DCMAKE_LIBRARY_PATH="$TP_INSTALL_DIR/lib64" \
```

Comment:
now we use a soft link  lib and lib64 are same dir， so there is no need to specify DCMAKE_LIBRARY_PATH, you can clean your `installed` dir and recompile again

Reply:
I build thirdparty in ubuntu, and I find that if i don't set CMAKE_LIBRARY_PATH，it would be failed

Reply:
https://github.com/apache/incubator-doris/pull/5394/files 
may be cmake found a wrong lib from system , you could reference this PR

Reply:
thank you for your reply @stdpain